Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -0,0 +1,166 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+/// Updates current user data to the backend and updates local storage.
+class CurrentUserUpdater<ExtraData: ExtraDataTypes>: Worker {
+    /// Load the most recent current user data.
+    ///
+    /// - Parameters:
+    ///   - currentUserId: The current user identifier.
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func loadUser(_ currentUserId: UserId, completion: ((Error?) -> Void)? = nil) {
```

Comment:
I thought we agreed we're not going to make this call since the local data are always updated.

Reply:
Discussed on a call, will change to updateDevices() :)